haas/cli.py

```diff
@@ -806,15 +739,13 @@ def show_console(node):
 @cmd
 def start_console(node):
     """Start logging console output from <node>"""
-    url = object_url('node', node, 'console')
-    do_put(url)
+    q = check_clientlib_response(lambda: C.node.start_console(node))
```

Comment:
Don't store the return value in a variable unless you're actually going to use it. same thing below.